core/src/main/resources/lib/hudson/propertyTable.jelly

```diff
@@ -63,14 +66,20 @@ THE SOFTWARE.
           <td style="white-space: normal" class="property-table__value">
             <j:choose>
               <j:when test="${attrs.sensitive}">
-                <div class="hidden-info">
-                  <div class="jenkins-button">
-                    ${%reveal}
-                  </div>
+                <div class="app-hidden-info-reveal">
+                  <button class="jenkins-button jenkins-button--transparent">
+                    <span class="jenkins-!-color-blue"><l:icon src="symbol-view"/> ${%reveal}</span>
```

Comment:
Once https://github.com/jenkinsci/jenkins/pull/6799 is in the class `jenkins-!-color-blue` could be moved to the button, that way the button hover background will be tinted blue too.